docs/geedocs/answer/6078762.html

```diff
@@ -67,7 +67,7 @@
         <tr>
           <td>17725523</td>
           <td>When attempting to install GEE 5.1.0 on RHEL 6 using the GUI installer, the process is likely to fail.</td>
```

Comment:
These are just release notes for old versions.  I suppose we could remove them, but I mostly left them as is.